collections/editor/includes/imgprocessor.php

```diff
@@ -1,9 +1,17 @@
 <?php
 if($LANG_TAG != 'en' && file_exists($SERVER_ROOT.'/content/lang/collections/editor/includes/imgprocessor.'.$LANG_TAG.'.php')) include_once($SERVER_ROOT.'/content/lang/collections/editor/includes/imgprocessor.'.$LANG_TAG.'.php');
 else include_once($SERVER_ROOT.'/content/lang/collections/editor/includes/imgprocessor.en.php');
+include_once($SERVER_ROOT.'/includes/head.php');
 ?>
 
 <script>
+
+	$(document).ready(function() {
```

Comment:
Is there a way to do this without jQuery?
Also, the left attribute _stil_ doesn't seem to be quite enough on my screen.